src/extensions/extension-loader.ts

```diff
@@ -86,20 +86,25 @@ export class ExtensionLoader {
   protected autoInitExtensions(register: (ext: LensExtension) => Function[]) {
     return reaction(() => this.toJSON(), installedExtensions => {
       for (const [extId, ext] of installedExtensions) {
-        let instance = this.instances.get(extId);
-        if (ext.isEnabled && !instance) {
+        const alreadyInit = this.instances.has(extId)
+
+        if (ext.isEnabled && !alreadyInit) {
           try {
-            const LensExtensionClass: LensExtensionConstructor = this.requireExtension(ext)
-            if (!LensExtensionClass) continue;
-            instance = new LensExtensionClass(ext);
+            const LensExtensionClass = this.requireExtension(ext)
+            if (!LensExtensionClass) {
+              continue;
+            }
+
```

Comment:
Redundant space? Or is it intentional?

Reply:
Intentional, I like spaces after blocks as it is good eye candy for reading code. Helps with grouping similar lines, sort of like spaces between paragraphs.

Reply:
Alright, but imho there is enough space since you've added `\n}` after `continue;`

Reply:
The block counts as well... We don't need to save on whitespace here... Is there something that you don't like about spreading out each step within a function?

Reply:
Didn't get what you mean about "spreading out each step"